hw/ip/csrng/doc/_index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "CSRNG HWIP Technical Specification"
+---
+
+# Overview
+
+This document specifies the Cryptographically Secure Random Number Generator (CSRNG) hardware IP functionality.
+Due to the importance of secure random number generation (RNG), it is a topic which is extensively covered in security standards.
+This IP targets compliance with both [BSI’s AIS31 recommendations for Common Criteria](https://www.bsi.bund.de/SharedDocs/Downloads/DE/BSI/Zertifizierung/Interpretationen/AIS_31_Functionality_classes_for_random_number_generators_e.pdf), as well as [NIST's SP 800-90](https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-90Ar1.pdf) (as referenced in [FIPS 140-3](https://csrc.nist.gov/publications/detail/fips/140/3/final)).
+Since these two standards use significantly different terminology, it is recommended that the reader refer to our RNG compliance strategy document for an overview of the various RNG classes and equivalencies between the two standards.
+The CSRNG IP supports both of these standards for both deterministic (DRNG) and true random number generation (TRNG).
+In NIST terms, it works with the entropy_source IP to satisfy the requirements as a DRBG or NRBG (non-deterministic random bit generator).
```

Comment:
```suggestion
In NIST terms, it works with the [Entropy Source IP]({{< relref "hw/ip/entropy_src/doc" >}}) to satisfy the requirements as a DRBG or NRBG (non-deterministic random bit generator).
```

(Please double-check if the link renders correctly.)

Reply:
Yes will do, thanks for the syntax help!

Reply:
Worked right the first time.  Very nice!